client/coral-admin/src/routes/Moderation/containers/Moderation.js

```diff
@@ -212,7 +212,7 @@ const withModQueueQuery = withQuery(gql`
     }) {
       ...CoralAdmin_Moderation_CommentConnection
     }
-    assets: assets(query: {}) {
+    assets: assets(query: {limit: 10}) {
```

Comment:
<img width="478" alt="screen shot 2017-06-16 at 3 44 53 pm" src="https://user-images.githubusercontent.com/1401559/27240458-26842792-52ab-11e7-838a-b7fd076ca3bc.png">

That one too :(